### PR TITLE
Fold the note on PCRE comments

### PR DESCRIPTION
**Pull request by kamil-tekiela**

### CODE REVIEW

File: reference/pcre/pattern.syntax.xml [hunk 1]
```diff
@@ -2161,6 +2161,41 @@
...
+$match = preg_match('/te(?# this is a comment)st/', $subject);
+var_dump($match);
+
+/* Whitespace and # is treated as part of the pattern unless PCRE_EXTENDED is enabled */
+$match = preg_match('/te   #~~~~
+st/', $subject);
+var_dump($match);
+
+/* When PCRE_EXTENDED is enabled, all whitespace data characters and anything
+   that follows an unescaped # in the pattern is ignored */
```
Comment (cmb69) [inline reference/pcre/pattern.syntax.xml:2183]: Maybe:
```suggestion
   that follows an unescaped # on the same line is ignored */
```

**Review by cmb69 — COMMENTED** [review]
Thank you! Generally good, but the meaning of `#` might need clarification.

**Comment by kamil-tekiela** [comment]
>the meaning of # might need clarification.
Is the text in the paragraph leading to the example not enough? What exactly do you have in mind?

**Review by cmb69 — APPROVED** [review]
> What exactly do you have in mind?

I was just referring to my suggestion. :)

**Pull request merged**